### ISSUE TITLE
Macros in props files don't resolve properly

### ISSUE THREAD

**Issue by s-martin** [Caphyon/clang-power-tools#172]
I have the issue that clang(-tidy) doesn't find some includes from external libraries.

We use .props files files heavily to add external libraries and these .props files use also macros for path substitution.

Example of a props file we use:
```
<?xml version="1.0" encoding="utf-8"?> 
<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
  <ImportGroup Label="PropertySheets" />
  <PropertyGroup Label="UserMacros" >
	<MYLIB_DIR>C:\Libraries\MYLIB\1.2.0.0-VS2017\</MYLIB_DIR>
	<!-- Map application Configurations to MYLIB configurations -->
	<MYLIBCONFIG Condition="'$(Configuration)'=='Debug'">Debug</MYLIBCONFIG>
	<MYLIBCONFIG Condition="'$(Configuration)'=='DebugConfig2'">Debug</MYLIBCONFIG>
	<MYLIBCONFIG Condition="'$(Configuration)'=='Release'">RelWithDebInfo</MYLIBCONFIG>
	<MYLIBCONFIG Condition="'$(Configuration)'=='ReleaseConfig2'">RelWithDebInfo</MYLIBCONFIG>
  </PropertyGroup>
  <PropertyGroup>
	<IncludePath>$(MYLIB_DIR)include\;$(IncludePath)</IncludePath>
    <LibraryPath>$(MYLIB_DIR)lib\$(MYLIBCONFIG)\;$(LibraryPath)</LibraryPath>
  </PropertyGroup>
  <ItemDefinitionGroup />
  <ItemGroup />
</Project>
```

As far as I understood clang-power-tools supports macros and .props files, but I suspect that maybe these configuration macros are not resolved properly.

**Comment by cpp-red-lion** [comment]
Hi,

Indeed some cases macros in props files were not expanded properly.
This issue has already been fixed and we're testing the changes.
Our next version (with these fixes) should be published this week.

We'll let you know.
Thanks for the feedback 👍

**Comment by cpp-red-lion** [comment]
This issue was fixed in `version 2.4` released today.
Thanks for you feedback 👍

**Comment by s-martin** [comment]
I can confirm this works for my setup.

Thanks for fixing this so quickly.
